docker/Dockerfile

```
@@ -126,6 +126,16 @@ RUN CACHE_STORE=array php -d memory_limit=512M artisan event:cache --no-interact
     CACHE_STORE=array php -d memory_limit=512M artisan icons:cache --no-interaction --no-ansi && \
     CACHE_STORE=array php -d memory_limit=512M artisan filament:cache-components --no-interaction --no-ansi
 
+# Download RoadRunner binary for Laravel Octane
+RUN curl -sSL https://github.com/roadrunner-server/roadrunner/releases/download/v2024.2.1/roadrunner-2024.2.1-linux-amd64.tar.gz \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)